### PR TITLE
[build] Makefile.am: add rule for src/bitcoin-wallet

### PR DESCRIPTION
**Pull request by Sjors**
Otherwise `make src/bitcoin-wallet` will fail with `No rule to make target`.

Also adds `bitcoin-wallet.exe` to the Windows installer.

### CODE REVIEW

**Comment by Sjors** [comment]
I just duplicated everything to make it consistent with `make src/bitcoin-tx`.

@jnewbery: I think this was missing from #13926

@ken2812221 does the Windows installer bit look OK? @MarcoFalke this probably needs a Gitian build.
 
Thanks @luke-jr.

**Comment by laanwj** [comment]
LGTM utACK https://github.com/bitcoin/bitcoin/pull/15388/commits/c9741df229652ff1ea2a14d705a8fac727d6cd55

**Comment by jnewbery** [comment]
utACK c9741df229652ff1ea2a14d705a8fac727d6cd55. Thanks @Sjors!

**Comment by promag** [comment]
Tested ACK c9741df on macOS.

File: Makefile.am [hunk 1]
```diff
@@ -76,6 +77,7 @@ $(BITCOIN_WIN_INSTALLER): all-recursive
 	STRIPPROG="$(STRIP)" $(INSTALL_STRIP_PROGRAM) $(BITCOIN_QT_BIN) $(top_builddir)/release
 	STRIPPROG="$(STRIP)" $(INSTALL_STRIP_PROGRAM) $(BITCOIN_CLI_BIN) $(top_builddir)/release
 	STRIPPROG="$(STRIP)" $(INSTALL_STRIP_PROGRAM) $(BITCOIN_TX_BIN) $(top_builddir)/release
+	STRIPPROG="$(STRIP)" $(INSTALL_STRIP_PROGRAM) $(BITCOIN_WALLET_BIN) $(top_builddir)/release
```
Comment (luke-jr) [inline Makefile.am]: You also need to add it to `share/setup.nsi.in`

File: Makefile.am [hunk 2]
```diff
@@ -86,7 +88,7 @@ $(OSX_APP)/Contents/PkgInfo:
 
 $(OSX_APP)/Contents/Resources/empty.lproj:
 	$(MKDIR_P) $(@D)
-	@touch $@ 
+	@touch $@
```
Comment (luke-jr) [inline Makefile.am]: Why the unrelated whitespace changes?

File: Makefile.am [hunk 3]
```diff
@@ -300,4 +305,3 @@ clean-docs:
 clean-local: clean-docs
 	rm -rf coverage_percent.txt test_bitcoin.coverage/ total.coverage/ test/tmp/ cache/ $(OSX_APP)
 	rm -rf test/functional/__pycache__ test/functional/test_framework/__pycache__ test/cache share/rpcauth/__pycache__
-
```
Comment (luke-jr) [inline Makefile.am]: More whitespace

**Review by luke-jr — CHANGES_REQUESTED** [review]

File: Makefile.am (hunk 2 above)
Comment (Sjors) [reply Makefile.am]: My editor is set to pedantic mode. Not sure why we missed this in #13753.

**Comment by Sjors** [comment]
Also added to `share/setup.nsi.in`.

**Comment by Sjors** [comment]
I added dbd65c2 to see if busting AppVeyor cache (again) can help with breaking builds in master. It's otherwise unrelated to this PR so I can drop the commit or move it to another PR later if needed.

**Comment by Sjors** [comment]
I moved the AppVeyor commits to a new PR #15405.

No amount of cache busting makes this AppVeyor error go away.

**Review by luke-jr — APPROVED** [review]
utACK
